src/components/timeline/index.js

```diff
@@ -84,6 +84,10 @@ type Props = {|
   ...ConnectedProps<{||}, StateProps, DispatchProps>,
 |};
 
+type State = {
```

Comment:
In our Flow configuration, our objects aren't exact objects by default. In general it's safer to make them exact:

https://flow.org/en/docs/types/objects/#toc-exact-object-types

Would you mind changing that here, that'll help ensure we don't do something silly in the future.